sdk-rs/src/types.rs

```diff
@@ -196,6 +196,10 @@ pub enum SdkError {
     InvalidBase58,
     #[error("insufficient SOL balance for fees")]
     OutOfSOL,
+    #[error("WebSocket connection failed")]
+    ConnectionError,
+    #[error("Couldn't subscribe to orderbook")]
+    SubscriptionFailure,
```

Comment:
we should surface the underlying error for this, something like
```suggestion
    #[error("Couldn't subscribe to orderbook: {0}")]
    SubscriptionFailure(#[from] tokio_tungestenite::Error),
```